### PR TITLE
commit_2

### PR DESCRIPTION
**Pull request by maserionius**

### CODE REVIEW

**Comment by maserionius** [comment]
whatch this

File: src/App.css [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-/*}*/
+.disliked{
+  width: 20%;
+  height: 20%;
+}
 
+.liked{
+  width:20%;
+  height: 20%;
+}
```
Comment (serh-ai) [inline src/App.css:4]: why liked and disliked are same? please refactor to same class

File: src/App.css [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
 .App {
-  /*text-align: center;*/
-  /*justify-content: center;*/
   vertical-align: middle;
   text-align: center;
   background: #66ffda;
 }
 
 .dncinGirl {
-  width:300px;
```
Comment (serh-ai) [inline src/App.css:22]: please use normal naming, like mainImage

File: src/App.js [hunk 3]
```diff
@@ -24,7 +22,7 @@ function App() {
                             setIndex(index + 1)
                             // setLiked(liked + 1)
                             liked.push(images[index])
-                            console.log('liked', liked)
+                            console.log('liked', liked, liked.length)
```
Comment (serh-ai) [inline src/App.js]: please do not use console log when ready

File: src/components/end.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export const End = ({liked, disliked}) => {
+    return (
+        <div><div>liked:</div>
```
Comment (serh-ai) [inline src/components/end.js]: please format to separate line

File: src/components/end.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    return (
+        <div>
+            <div>liked:</div>
+            {liked.map(function (stroka) {
+                return <img src={stroka} className='image_final_list'/>
+            })}
+            <div>disliked:</div>
+            {disliked.map(function (stroka) {
+                return <img src={stroka} className='image_final_list'/>
+            })}
```
Comment (serh-ai) [inline src/components/end.js]: please extract to separate component, like `<PhotoList images={images}>`, also use separate css for them

**Review by serh-ai — APPROVED** [review]

**Pull request merged**